comparisons/mcnemar/README.md

```diff
@@ -17,14 +17,58 @@ tags:
 
 ## Comparison description
 
+McNemar's test is a non-parametric diagnostic test over a contingency table resulting from the predictions of two classifiers. The test compares the sensitivity and specificity of the diagnostic tests on the same group reference labels. It can be computed with:
+
+McNemar = (SE - SP)**2 / SE + SP
```

Comment:
Could we connect SE/SP to predictions1/2 and references? Just looking at this formula I would think you can just use SE/SP of a single model's prediction.